apps/condo/domains/contact/components/EditContactForm.tsx

```diff
@@ -110,11 +109,11 @@ export const EditContactForm: React.FC = () => {
                 {
                     ({ handleSave, isLoading }) => {
                         return (
-                            <Row>
-                                <Col span={3}>
+                            <Row gutter={[0, 40]} justify={'center'}>
```

Comment:
Please move gutter to a constant

```
const AVATAR_GUTTER = [0, 40]
...
<Row gutter={AVATAR_GUTTER} justify={'center'}>
```

Reply:
this rule should be fixed in code convention, i guess)